### PR TITLE
docs: add more advanced examples

### PR DESCRIPTION
**Pull request by yuuzhan**
Added a monthly option to auto backup advanced examples.  I also added compressed=true to the default examples :P

### CODE REVIEW

**Comment by jcwillox** [comment]
Compressed already defaults to true right? so we probably don't need to include it in the examples

**Comment by yuuzhan** [comment]
Correct, I did not realize that when I had made the edits :)  Just did a test however and it is still compressed without the parameter set.

**Comment by yuuzhan** [comment]
I just added the code that I had been using to make use of the new Blocks for automation (allowing all your backups to be in one automation).

**Comment by jcwillox** [comment]
Great, looks good to me, only this is looks like you duplicated the `Full backup on the 1st of every month` example

File: docs/advanced-examples.md [hunk 1]
```diff
@@ -91,3 +91,86 @@ Of course, you can tweak these values to your liking, or even add a month/yearly
...
+    value_template: "{{ now().day == 1 }}"
+action:
+  - service: auto_backup.backup_full
+    metadata: {}
+    data:
+      name: "MonthlyBackup: {{ now().strftime('%A, %B %-d, %Y') }}"
+      # store backup for a year
+      # i.e. backup on the 1st, store for 12 months
+      keep_days: 365
+```
```
Comment (jcwillox) [inline docs/advanced-examples.md:133]: this looks to be duplicated above

**Review by jcwillox — APPROVED** [review]

File: docs/advanced-examples.md (hunk 1 above)
Comment (yuuzhan) [reply docs/advanced-examples.md]: Whoops!

**Review by jcwillox — APPROVED** [review]

**Pull request merged**